### DISCUSSION TITLE
aliyundrive-webdav is featured in Moka's README (Moka in Production).

### DISCUSSION THREAD

**Discussion by tatsuya6502** [messense/aliyundrive-webdav#207] [Show and tell]
@messense — Just wanted to let you know that I have added "Moka in Production" section to Moka's README, and wrote about your aliyundrive-webdav.

- https://github.com/moka-rs/moka#moka-in-production

I hope you will like it. If you have any issues, please let me know. (PRs for the README are welcome)

Thank you for using Moka!

**Comment by messense** [comment]
Thank you! Looks great!
